### PR TITLE
Add option to specify custom files to ignore in watch mode

### PR DESCRIPTION
**Pull request by jozefcipa**
#### Description of what you did:
This PR adds the option to specify custom files to ignore when running `yarn develop` or `yarn develop --watch-admin`.
Example of using:
```
// ./config/environments/development/server.json
"admin": {
    "watchIgnoreFiles": [
        "**/docker",
        "**/docker/**"
     ]
}
```
Now, after applying this the server won't be restarted when something in `./docker` folder changes (e.g. database writes)

#### My PR is a:

- [ ] 💥 Breaking change
- [ ] 🐛 Bug fix
- [x] 💅 Enhancement
- [ ] 🚀 New feature

#### Main update on the:

- [ ] Admin
- [ ] Documentation
- [x] Framework
- [ ] Plugin

#### Manual testing done on the following databases:

- [x] Not applicable
- [ ] MongoDB
- [ ] MySQL
- [ ] Postgres
- [ ] SQLite

### CODE REVIEW

**Comment by jozefcipa** [comment]
Sure, it's there. @alexandrebodin @derrickmehaffy

File: docs/3.0.0-beta.x/concepts/configurations.md [hunk 1]
```diff
@@ -450,6 +450,7 @@ The session doesn't work with `mongo` as a client. The package that we should us
 - `admin`
   - `autoOpen` (boolean): Enable or disabled administration opening on start (default: `true`)
   - `path` (string): Allow to change the URL to access the admin (default: `/admin`).
+  - `watchIgnoreFiles` (array): Add custom files that should not be watched during development (default: `[]`)
```
Comment (alexandrebodin) [inline docs/3.0.0-beta.x/concepts/configurations.md]: can you just link to the chokidar do for the pattern documentation ;)

Comment (jozefcipa) [reply docs/3.0.0-beta.x/concepts/configurations.md]: Do you think of anything particular or is it enough to just provide this link https://github.com/paulmillr/chokidar? I couldn't find docs :/ @alexandrebodin

Comment (alexandrebodin) [reply docs/3.0.0-beta.x/concepts/configurations.md]: You should link to this https://github.com/paulmillr/chokidar#path-filtering
specifying it uses the ignored param of chokidar :)

Comment (jozefcipa) [reply docs/3.0.0-beta.x/concepts/configurations.md]: take a look now, please :)

**Pull request merged**